spec/rspec/runner_spec.rb

```diff
@@ -106,11 +118,9 @@
     context 'with expired map' do
       before { allow(prediction_builder).to receive(:expired_map?).and_return true }
 
-      let(:out_stream) { double(puts: true) }
-
       it 'prints out warning' do
-        expect(out_stream).to receive(:puts).with('Maps are outdated!')
-        described_class.run([], STDERR, out_stream)
+        expect(Crystalball::Logger).to receive(:warn).with('Maps are outdated!')
```

Comment:
your stubs are inconsistent [here](https://github.com/toptal/crystalball/pull/88/files#diff-21ff99f1c30317f4701f73bebb218093R100) and [here](https://github.com/toptal/crystalball/pull/88/files#diff-21ff99f1c30317f4701f73bebb218093R27) you stub it as `Crystalball::Logger.instance`

Reply:
That's because `Logger.configure` does `instance_eval` on the instance, so it's the instance that receives the `log_level=` for example. I think I can change it all to expect on the instance, for the sake of consistency.